pdf_parser/pdf_parse.py

```diff
@@ -81,7 +80,7 @@ def parse_pdf_document(document):
     pdf_file = PdfFile(file_pages)
     html_file.close()
     os.remove(parsed_path)
-    return pdf_file, text
+    return pdf_file
```

Comment:
Does this mean that the PDF text won't be pushed into the JSON files we produce?